src/components/RenderPageRoute.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Route } from "react-router-dom";
+import { makeStyles } from "@material-ui/core/styles";
+
+import type { AppPage } from "../app-config.types";
+
+const useStyles = makeStyles((theme) => ({
+  outerPageDiv: {
+    zIndex: 1095,
+    backgroundColor: "white",
+    width: "100%",
+    height: "100%",
+    boxSizing: "border-box",
+    flexShrink: 0,
+    overflow: "hidden",
+    paddingTop: "64px",
+    [theme.breakpoints.down("xs")]: {
+      // on mobile devices
+      paddingTop: "0 !important",
```

Comment:
!important can be removed. I added it for testing, but it also works without.

Reply:
Ok, mach ich nachher noch.